modules/ROOT/pages/deployment/general/general-info.adoc

```diff
@@ -159,39 +164,72 @@ Because Infinite Scale does not use a database for storing information like user
+
+// ocis/services$ grep -r 'OCIS_BASE_DATA_PATH' --include=*.go **/pkg/config
+
+* xref:{s-path}/auth-basic.adoc[OCIS_LDAP_CACERT;AUTH_BASIC_LDAP_CACERT, window=_blank]
+* xref:{s-path}/graph.adoc[OCIS_LDAP_CACERT;GRAPH_LDAP_CACERT, window=_blank]
+* xref:{s-path}/groups.adoc[OCIS_LDAP_CACERT;GROUPS_LDAP_CACERT, window=_blank]
+* xref:{s-path}/idm.adoc[IDM_LDAPS_CERT, window=_blank]
+* xref:{s-path}/idm.adoc[IDM_LDAPS_KEY, window=_blank]
+* xref:{s-path}/idm.adoc[IDM_DATABASE_PATH, window=_blank]
+* xref:{s-path}/idp.adoc[IDP_ENCRYPTION_SECRET_FILE, window=_blank]
+* xref:{s-path}/idp.adoc[IDP_SIGNING_PRIVATE_KEY_FILES, window=_blank]
+* xref:{s-path}/idp.adoc[IDP_TRANSPORT_TLS_CERT, window=_blank]
+* xref:{s-path}/idp.adoc[IDP_TRANSPORT_TLS_KEY, window=_blank]
+* xref:{s-path}/nats.adoc[NATS_NATS_STORE_DIR, window=_blank]
+* xref:{s-path}/nats.adoc[NATS_TLS_CERT, window=_blank]
+* xref:{s-path}/nats.adoc[NATS_TLS_KEY, window=_blank]
+* xref:{s-path}/ocm.adoc[OCM_OCM_INVITE_MANAGER_JSON_FILE, window=_blank]
+* xref:{s-path}/ocm.adoc[OCM_OCM_PROVIDER_AUTHORIZER_PROVIDERS_FILE, window=_blank]
+* xref:{s-path}/ocm.adoc[OCM_OCM_CORE_JSON_FILE, window=_blank]
+* xref:{s-path}/ocm.adoc[OCM_OCM_SHAREPROVIDER_JSON_FILE, window=_blank]
+* xref:{s-path}/proxy.adoc[PROXY_TRANSPORT_TLS_KEY, window=_blank]
+* xref:{s-path}/search.adoc[SEARCH_ENGINE_BLEVE_DATA_PATH, window=_blank] ^*^
+* xref:{s-path}/settings.adoc[SETTINGS_DATA_PATH, window=_blank]
+* xref:{s-path}/sharing.adoc[SHARING_USER_JSON_FILE, window=_blank]
+* xref:{s-path}/sharing.adoc[SHARING_PUBLIC_JSON_FILE, window=_blank]
+* xref:{s-path}/store.adoc[STORE_DATA_PATH, window=_blank]
+* xref:{s-path}/storage-system.adoc[STORAGE_SYSTEM_OCIS_ROOT, window=_blank]
+* xref:{s-path}/storage-users.adoc[STORAGE_USERS_OCIS_ROOT, window=_blank]
+* xref:{s-path}/storage-users.adoc[STORAGE_USERS_S3NG_ROOT, window=_blank]
+* xref:{s-path}/storage-users.adoc[STORAGE_USERS_OWNCLOUDSQL_DATADIR, window=_blank]
+* xref:{s-path}/storage-users.adoc[STORAGE_USERS_OWNCLOUDSQL_UPLOADINFO_DIR, window=_blank]
+* xref:{s-path}/thumbnails.adoc[THUMBNAILS_FILESYSTEMSTORAGE_ROOT, window=_blank] ^*^
+* xref:{s-path}/users.adoc[OCIS_LDAP_CACERT;USERS_LDAP_CACERT, window=_blank]
 
 // necessary to properly break the list item from the next component
 {empty}
 
-.The default locations for the _base_ directory are:
-{empty}
---
-* For container images (inside the container) +
+(*) ... Note that it is recommended for these services to configure an individual path due to expected higher data quantities on larger installations, see the important note on possible impacts below.
+
+==== Default Location
+
+* The base path has a default location for metadata and service dependent data (xref:#services_using_base_path[see above]) which *must be* on a POSIX storage. If not otherwise defined via xref:using-s3-for-blobs[Using S3 for Blobs], it is also used storing blobs using that path:
```

Comment:
```suggestion
* The base path has a default location for metadata and service dependent data (xref:#services_using_base_path[see above]) which *must be* on a POSIX storage. If not otherwise defined via xref:using-s3-for-blobs[Using S3 for Blobs], it is also used to store blobs using that path:
```